packages/core/src/native-reserved-lists.ts

```diff
@@ -102,86 +102,81 @@ export const nativePseudoElements = [
     'spelling-error',
 ];
 
-export const nativeFunctionsDic = {
-    annotation: true,
-    attr: true,
-    blur: true,
-    brightness: true,
-    calc: true,
-    'character-variant': true,
-    circle: true,
-    clamp: true,
-    'conic-gradient': true,
-    constant: true,
-    contrast: true,
-    counter: true,
-    counters: true,
-    'cubic-bezier': true,
-    'drop-shadow': true,
-    ellipse: true,
-    env: true,
-    'fit-content': true,
-    format: true,
-    grayscale: true,
-    hsl: true,
-    hsla: true,
-    'hue-rotate': true,
-    hwb: true,
-    image: true,
-    inset: true,
-    invert: true,
-    leader: true,
-    'linear-gradient': true,
-    local: true,
-    matrix: true,
-    matrix3d: true,
-    max: true,
-    min: true,
-    minmax: true,
-    opacity: true,
-    ornaments: true,
-    paint: true,
-    path: true,
-    perspective: true,
-    polygon: true,
-    'radial-gradient': true,
-    rect: true,
-    repeat: true,
-    'repeating-linear-gradient': true,
-    'repeating-radial-gradient': true,
-    'repeating-conic-gradient': true,
-    rgb: true,
-    rgba: true,
-    rotate: true,
-    rotate3d: true,
-    rotateX: true,
-    rotateY: true,
-    rotateZ: true,
-    saturate: true,
-    scale: true,
-    scale3d: true,
-    scaleX: true,
-    scaleY: true,
-    scaleZ: true,
-    sepia: true,
-    skew: true,
-    skewX: true,
-    skewY: true,
-    steps: true,
-    styleset: true,
-    stylistic: true,
-    swash: true,
-    symbols: true,
-    translate: true,
-    translate3d: true,
-    translateX: true,
-    translateY: true,
-    translateZ: true,
-    url: true,
-    var: true,
+export const nativeFunctionsDic: Record<string, { preserveQuotes: boolean }> = {
```

Comment:
is it public? I would keep another dict with this info. no need to change this record.

Reply:
This should have been a Set anyway

Reply:
Not public, used only in this specific one place. 

I would change the behavior to always preserve whatever was in the source for native value functions, but that is a bigger change that I just don't have the time to check for atm.

We need to organize all the "native/spec" at some point and also refactor the "functions" module.